x/auth/vesting/types/types.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+package cosmos_sdk.x.auth.vesting.v1;
+
+import "third_party/proto/gogoproto/gogo.proto";
+import "types/types.proto";
+import "third_party/proto/cosmos-proto/cosmos.proto";
+import "x/auth/types/types.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/auth/vesting/types";
+
+// Account defines the default concrete vesting account type used in the x/auth
+// module.
+//
+// This type should only be used internally for testing purposes. The application's
+// concrete Account type should be defined in the application-level codec.
+message VestingAccount {
```

Comment:
Maybe name to something like `TestVestingAccount` just so it isn't used mistakenly.